### ISSUE TITLE
Missing run configuration template

### ISSUE THREAD

**Issue by sorenmarkert** [Janhouse/sap-commerce-integration-intellij-idea-plugin#1]
I just tried installing this pluing instaed of the paid version. And after re-importing the project, the template to create Hybris tests was gone.
Could you please add them?

**Comment by Janhouse** [comment]
Thanks, I noticed something about it. IntelliJ even shows a suggestion to install the paid version to support that feature. I will look into it.

**Comment by Janhouse** [comment]
Looks like they were silently moving towards the pro version from 2017 without letting others know.
Their previous free versions in the JetBrains repo contain some additional functionality that is not in the LGPL licensed code.

**Comment by sorenmarkert** [comment]
So all development since 2017 is missing?

**Comment by Janhouse** [comment]
@sorenmarkert, no, only some features seem to be missing. I started outlining those in the plugin description ( https://plugins.jetbrains.com/plugin/12639-integration-for-sap-commerce ).
Seems like they did publish all of the bug repository fixes publicly and then probably all the things that related to fixing those.
And as they distributed compiled plugin through JetBrains without providing source (though no one asked, maybe they would have provided the source if someone asked), they probably broke the terms of their own chosen LGPL 3.0 license unless they got permission from all of the contributors to relicense to proprietary license. But then again - it was always listed as LGPL 3.0 licensed code in the plugin repository.

**Comment by AlexanderBartash** [comment]
@Janhouse Let me help you. This repository contains not just bugfixes. And no we did not break LGPL. If you pay attention you will notice that the Pro version uses the LGPL version as a library, which is the whole sense of LGPL Also I am the solely owner of all rights of all code since the day 1 the hard condition of accepting any contribution was giving a sign-off for the rights of the code you provide.

And no it was not always listed as LGPL. The plugin repo does not say anything about LGPL for years.

**Comment by Janhouse** [comment]
@AlexanderBartash, ah, thanks for the explanation and your contribution to the open source community!

I just want to correct you - the "sign-off" is not an agreement to give the code away. It is simply a note on the commit that identifies which person checked and can vouch for the code quality and that it doesn't break the license.  It just "legally" puts responsibility on the person who signed it. The origin of this was with lawsuit against Linux some time ago. If you think that this gave you any rights to that code you are very very wrong.
And I did check all of the files in the open source repository and there was absolutely nothing mentioning giving away of rights.

And I guess it nicely answers @sorenmarkert then. If the change was made in the open source library then it was pushed to open source repository but if it was used externally then it was not.

**Comment by AlexanderBartash** [comment]
@Janhouse When some new contributor wanted to contribute I've been asking people explicitly to to give away right for the code they provide. In any way LGPL permits use as a library.

Thank you too, I'd love to see this fork live on and evolve, but honestly after 5 years of maintaining it at my expense and almost no one caring I can not continue doing this. I honestly wish you good luck!

**Comment by Janhouse** [comment]
@AlexanderBartash I will probably do the minimum to keep it working with upcoming IntelliJ IDEA versions for some time, but I don't plan on adding anything too cool to it since existing version is good enough as it is right now. Will see how SAP Commerce Cloud v2 changes it in the upcoming years.

And if your version introduces something very desirable and pricing changes drastically then I could see many companies buying it. But if the pricing stays the same as it is now, then it is probably cheaper in the long term to implement similar features into this fork, at least for the company I work for.
If it was one time payment with some sort of upgrade payments for large updates then the company I work for would have bought it without question but as it is right now we don't think it is worth paying more than 10K EUR yearly for a plugin that doesn't change that much.

Also, if you had actively approached some of the larger IT companies with Hybris teams, you probably could have gained decent amount of money for the plugin years ago.

Nonetheless you will probably do good enough in Western European and USA market already now. I will be keeping an eye out for features in your plugin. 😃

**Comment by AlexanderBartash** [comment]
@Janhouse I can reassure you that I have tried everything possible to make big corporations pay for this. Unfortunately it did not work. I understand the problems many companies face but this plugin is also some sort of a company and it needs income to sustain itself.

**Comment by divyakumarjain** [comment]
Can we create a new github organization using https://github.com/organizations/new and manage the open source repository there?

This will also allow us to raise issue for all the items which are broken at central place in single repository.

**Comment by Janhouse** [comment]
@divyakumarjain not sure what would be the purpose of it since there is no organization. If some miracle happens and some active contributors appear, I'll give them write and merge rights to this repository. Meanwhile I'll monitor progress in other forks and go through pull requests if any are created.
I am maintaining some other open source projects this way in other people's Github repositories and it works fine.

**Comment by divyakumarjain** [comment]
Seems like there is another fork https://github.com/epam/sap-commerce-intellij-idea-plugin which is being maintained and worked upon.

Is there a way we can combine these forks into single forks so that we can together keep this plugin free and opensource.

Again the same idea which was proposed earlier of having single organization in Github without single owner of the repository. I have been part of ng-formly in github. it worked great.
